guides/release/object-model/computed-properties.md

```diff
@@ -49,33 +50,32 @@ Outside of those two circumstances the code in the property will not run, even i
 We'll modify the `fullName` property from the previous example to log to the console:
 
 ```javascript
-import Ember from 'ember';
+import EmberObject, { computed } from '@ember/object';
 
-…
```

Comment:
I decided to expand this to a complete example, so that the syntax highlighting is correct